src/godot-tools.ts

```diff
@@ -101,6 +104,17 @@ export class GodotTools {
 		vscode.env.clipboard.writeText(relative_path);
     }
 
+    private open_type_documentation(uri: vscode.Uri) {
+        // get word under cursor
+        let activeEditor = vscode.window.activeTextEditor;
+        let document = activeEditor.document;
+        let curPos = activeEditor.selection.active;
+        let wordRange = document.getWordRangeAtPosition(curPos);
+        let symbolName = document.getText(wordRange);
```

Comment:
Use `const` whenever possible:

```suggestion
        const activeEditor = vscode.window.activeTextEditor;
        const document = activeEditor.document;
        const curPos = activeEditor.selection.active;
        const wordRange = document.getWordRangeAtPosition(curPos);
        const symbolName = document.getText(wordRange);
```